std/math.d

```diff
@@ -8281,27 +8292,80 @@ if (isFloatingPoint!(F) && isFloatingPoint!(G))
     return impl(x, y);
 }
 
+// Due to issue 20451 this doesn't work on Win32 and FreeBSD. It would work
+// using isClose, but the main purpose of this test is to provide a good example,
+// not to test the pow function, and this would lead to a somewhat ugly example.
```

Comment:
The standard library documentation should not mislead users into thinking that floating-point operations are perfectly precise when they're not. Using `isClose`, `approxEqual`, or `feqrel` is the honest and correct option, even if it is less aesthetically pleasing.

Reply:
I'm not sure if you are aware of the transformations on the unittests, that are done by ddoc. Every `assert` is replaced by a `writeln`. But the output of `writeln` isn't compared anymore by a computer (but by the human reading the example), so the problem does not exist there. The strange comparisons can only be seen, when the user looks into the source code. And there they are next to this comment explaining what's going on.

I actually changed the comment and the versioning a little bit, because there might be other OSs which fail. Now this unittest is only used by ddoc. I hope this is sort of a compromise for you.